mmv1/products/accesscontextmanager/api.yaml

```diff
@@ -899,6 +899,196 @@ objects:
                 The list of APIs usable within the Service Perimeter.
                 Must be empty unless `enableRestriction` is True.
               item_type: Api::Type::String
+          - !ruby/object:Api::Type::Array
+            name: 'ingressPolicies'
+            description: /
+              List of `IngressPolicies` to apply to the perimeter. A perimeter may
+              have multiple `IngressPolicies`, each of which is evaluated
+              separately. Access is granted if any `Ingress Policy` grants it.
+              Must be empty for a perimeter bridge.
+            item_type: !ruby/object:Api::Type::NestedObject
```

Comment:
We likely need to mark some of the subfields here as `Required`, or at least mark some as `AtLeastOneOf`/`ExactlyOneOf`. Probably Required because this is inside an array which makes the other two options not viable

We try to do this for nested objects because allowing users to specify empty object often causes problems. Can we mark at least one field on every level of the object as Required?